.gitignore

```diff
@@ -15,3 +15,4 @@ logs
 /rsc/*.pref
 /rsc/*.xml
 /target/
+LOG_DEST/
```

Comment:
this should be a dev local folder and could be anything, I don't know if to add to .gitignore because of course should not be committed (whatever it is)